py-pkg/pkgnet/dependency_reporter.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional, Callable
+
+try:
+    import importlib.resources as importlib_resources
+except ImportError:
+    # Python 3.6 uses importlib_resources backport
+    import importlib_resources
+
+
+import pandas as pd
+import pkg_resources
+
+from pkgnet.abstract_graph_reporter import AbstractGraphReporter
+from pkgnet.abstract_package_reporter import registrar
+from pkgnet.graphs import DirectedGraph
+from pkgnet.search_utils import recursive_node_search
+
+
+@registrar.register_reporter
+class DependencyReporter(AbstractGraphReporter):
+
+    _graph_class = DirectedGraph
+
+    report_template = "tab_dependency_report.jinja"
+    report_slug = "dependency-report"
+    report_name = "Dependencies"
+    layout = "kamada_kawai_layout"
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
```

Comment:
I agree with only supporting `kwargs` and not adding `*args` here. That will help keep the code explicit